src/activitylog/activitylog.html

```diff
@@ -14,27 +18,38 @@ <h1>Extension Activity Monitor</h1>
       <div class="notice"></div>
       <div class="activity-log-wrapper">
         <h2>Activity Logs</h2>
-        <div class="activity-log-container">
-          <div class="log-table">
-            <table>
-              <thead>
-                <tr>
-                  <th>Extension Id</th>
-                  <th>Timestamp</th>
-                  <th>API Type</th>
-                  <th>API Name</th>
-                  <th>View Type</th>
-                </tr>
-              </thead>
-              <tbody></tbody>
-            </table>
-          </div>
-          <div class="show-log-detail-wrapper" hidden>
-            <div id="showLogDetails"></div>
-            <div class="close"></div>
-          </div>
-        </div>
+        <log-view></log-view>
       </div>
     </div>
+    <template class="log-table-template">
```

Comment:
There is only one template in the document, use the `id` attribute instead.

(also at the next template below).

And add a blank line before the `<template ...>` element to slightly improve readability.

Reply:
done :-)